src/Components/DisplayReveresQuestions/utils.ts

```diff
@@ -0,0 +1,58 @@
+import { workerData } from "worker_threads";
+import { Question } from "../../Apis/types";
+
+interface Props {
+    questions: Question[];
+}
+
+function reverseWords(str: string) {
+    const allWords = str.split(" ");
+    return allWords
+        .map((item: string) => item.split("").reverse().join(""))
+        .join(" ");
+}
+
+export const sortReverseOrderAndReverseString = ({ questions }: Props) => {
+    let reversedOrder = questions.sort((a, b) => {
+        if (a.question > b.question) return -1;
+        if (a.question > b.question) return 1;
+        return 0;
+    });
+
+    reversedOrder.forEach(
+        ({ question }) => (question = reverseWords(question))
```

Comment:
won't set the question as the reversed question.

Reply:
You want map, not for each

Reply:
It will return a new array that you set to a new variable